kernel/configs/fragments/x86_64/gpu/vfio.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_VFIO_NOIOMMU=y
```

Comment:
I dont see why you are enabling CONFIG_VFIO_NOIOMMU and  CONFIG_VFIO_PCI? 
Don't these conflict? You either want IOMMU or not.
Also, this config is going to taint the kernel: 
```
VFIO No-IOMMU mode enables IOMMU groups for devices without IOMMU backing for the purpose of re-using the VFIO infrastructure in a non-secure mode. Use of this mode will result in an unsupportable kernel and will therefore taint the kernel. 
```

Reply:
cc @egernst

Reply:
I confirm this is not required also. I willl remove it.